.changeset/hot-onions-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+'@chainlink/proof-of-reserves-adapter': minor
+'@chainlink/cryptoapis-adapter': minor
```

Comment:
```suggestion
'@chainlink/cryptoapis-adapter': minor
```

Reply:
Just to be sure, you want to remove `proof-of-reserves-adapter` even with the change in `packages/composites/proof-of-reserves/src/balance.ts`?

Reply:
Oooh, sorry didn't see that. This probably don't need to change then, thanks!